source/includes/documentation/_vue_js_elements.md

```diff
@@ -732,45 +732,78 @@ For examples of usage see: [JDI Vuetify Sliders tests](https://github.com/jdi-te
 
 #### 5.12.4 Switches
 
-[Vuetify documentation page](https://vuetifyjs.com/en/components/switches/)
+[Switches Vuetify documentation page](https://vuetifyjs.com/en/components/switches/)
 
+Switches are located in the following class:
 - __Java__: _com.epam.jdi.light.vuetify.elements.common.Switch.java_
 
 ```java
-    @Test
-public void colorTest() {
-  colorSwitches.forEach(Switch::isDisabled);
+@Test
+public void functionalityAndCssStyleSwitchTest() {
+  String red = "red";
+  final Switch redSwitch = colorSwitches.get(1);
+  redSwitch.show();
+  redSwitch.is().checked();
+  redSwitch.label().has().text(red);
+  redSwitch.has().value(red);
+  redSwitch.has().color(RED.value());
+  redSwitch.has().detailsHidden();
+  redSwitch.uncheck();
+  redSwitch.is().unchecked();
+}
+```
+__Switches__ - The `v-switch` component provides users the ability to choose between two distinct values.
 
-  colorSwitches.get(1).is().inputColor(Colors.RED.value());
-  colorSwitches.get(1).uncheck();
-  colorSwitches.get(1).is().inputColor(Colors.BLACK_TRANSPARENT_087.value());
+![Switches example](../../images/vuetify/switches.png)
 
-  colorSwitches.get(6).is().inputColor(Colors.ORANGE_DARKEN_3.value());
-  colorSwitches.get(6).uncheck();
-  colorSwitches.get(6).is().inputColor(Colors.BLACK_TRANSPARENT_087.value());
+__Vuetify v2.6.14__ code example:
 
-  colorSwitches.get(12).is().inputColor(Colors.RED_ACCENT_2.value());
-  colorSwitches.get(12).uncheck();
-  colorSwitches.get(12).is().inputColor(Colors.BLACK_TRANSPARENT_087.value());
-  }
+```html
+<div class="v-input v-input--hide-details theme--light v-input--selection-controls v-input--switch">
+  <div class="v-input__control">
+    <div class="v-input__slot">
+      <div class="v-input--selection-controls__input">
+        <input aria-checked="false" id="input-1187" role="switch" type="checkbox" aria-disabled="false" value="red">
+        <div class="v-input--selection-controls__ripple">
+        </div>
+        <div class="v-input--switch__track theme--light">
+        </div>
+        <div class="v-input--switch__thumb theme--light">
+          <!---->
+        </div>
+      </div>
+      <label for="input-1187" class="v-label theme--light" style="left: 0px; right: auto; position: relative;">red</label>
+    </div>
+  </div>
+</div>
```

Comment:
Sorry, my bad, did not notice at first. But it is not Vuetify code.
Shouldn't we have example of Vuetify template like this one?
It's a question :) Maybe you agreed with smb that it shoud be html?
`<template>
  <v-container
    class="px-0"
    fluid
  >
    <v-switch
      v-model="switch1"
      :label="`Switch 1: ${switch1.toString()}`"
    ></v-switch>
  </v-container>
</template>`